hash/gbcolor.xml

```diff
@@ -10504,6 +10897,22 @@ Unreleased (music source code exists, possibly no prototypes exist)
 		</part>
 	</software>
 
+	<software name="lionkingfa" cloneof="lionking">
+		<!-- Notes: GBC only -->
+		<!--	In lot check as "cgbblkf1.573"	-->
+		<description>Le Roi Lion - Les Adventures de Simba (Fra, Rev. A)</description>
```

Comment:
Does it really have a spelling error in the title?

Reply:
That's how it was in the softlist already, I didn't change it.

Reply:
Also, it does indeed: https://youtu.be/ugZlvC8UXfA?t=17

Reply:
I've added the (more correct?) title from the box as the "alt_title".